tests/jballerina-integration-test/src/test/java/org/ballerinalang/test/packaging/SingleBalFileTestCase.java

```diff
@@ -94,6 +94,19 @@ public void testSingleBalFileWithJDBCUsage() throws BallerinaTestException {
         fooRunLeecher.waitForText(10000);
     }
 
+    @Test(description = "Test running bal with relative path")
+    public void testRunWithRelativePath() throws BallerinaTestException {
+        String testBalFile = "hello_world.bal";
+        Path testRelativeFilePath = Paths.get("src", "test", "resources", "packaging", "singleBalFile",
+                "relative", "testDir");
+        // Run and see output
+        String msg = "Hello, World!";
+        LogLeecher fooRunLeecher = new LogLeecher(msg);
+        balClient.runMain("run", new String[]{"../testBal/hello_world.bal"}, new HashMap<>(), new String[0],
```

Comment:
I dont think we can hardcode the file separator.

Reply:
Used File.separator instead of hardcoding the path.